src/edit.js

```diff
@@ -81,6 +84,28 @@ useEffect(() => {
     })
 };*/
 
+const mappedUsers = [
+    // Set a default value when no user is selected.
+    {
+        value: 0,
+		label: 'Select a user…',
+		disabled: true,
+    },
+    // Spread the map of users into the array.
+    ...users.map(
+        ( user ) => {
+            return {
+                label: user.name,
+                value: user.id,
+                id: user.id,
+                name: user.name,
+                bio: user.description,
+                avatar: user.avatar_urls[ 96 ] ?? 'https://www.gravatar.com/avatar/?d=mystery-person'
+            };
+        }
+    )
+]
```

Comment:
I added a default value here, then spread the results of the `map` below.

Remember that spreading just unpacks an array into a new array. It's really useful shorthand!